app/server/methods/ratingsMethods.js

```diff
@@ -0,0 +1,68 @@
+Meteor.methods({
+  rateApplication: function(ratingNumber, applicationId) {
+    check(ratingNumber, Number);
+    check(applicationId, String);
+
+    if (!this.userId) {
+      throw new Meteor.Error('error-not-signed-in', 'You must sign in first.');
+    }
+
+    if (!Roles.userIsInRole(this.userId, ['committee',])) {
+      throw new Meteor.Error('not-allowed', 'You must be a committee user aka No Juice Error');
+    }
+
+    // Find existing rating
+    var existingRating = Ratings.findOne({userId: Meteor.userId(), applicationId: applicationId,});
+    
+    // Create rating update
+    var rating = {
```

Comment:
We could get rid of `rating` object and update `existingRating` with new values instead, right?